### ISSUE TITLE
Issues with this repo

### ISSUE THREAD

**Issue by receter** [federicoReghini/vite-lib-reanimated#1]
At this time the lib in this repo does not yet export anything in the main file.

If I run "npm run dev" I see issues related to `react-native` not beeing configured to work with vite.

Do you have a repo with a consumer of your library that breaks only if a component from the package is imported? That would help to find the issue.

![grafik](https://github.com/federicoReghini/vite-lib-reanimated/assets/2504695/fdaf46c6-c693-4b97-abf1-6b5ee5517dc5)

**Comment by federicoReghini** [comment]
Hi, Sorry didn't even notice. I didn't configure vite for react-native because i do not need to test my comps in the project( in the real lib). I'm gonna try to configure it

**Comment by federicoReghini** [comment]
hey @receter  i finally have updating for you. Sorry if it taken longer than i thought.
here is the [expo project with the library](https://github.com/federicoReghini/vite-lib-reanimated-app).
Right know inside tabs/two.tsx you have an animatedText imported from the library that is working fine(in my real project i'd already had problems here) and if you press the text the app than will crash.
When you press it you will see it try to use a method toast.success that fire a event that is passed to a provider that handle it (inside app/layout.tsx. It is animated and it makes the app crash.
this is more or less a real comp that i've in the real repo. but i also had this crash while trying to use  animated custom charts.
I'will investigate further.

**Comment by receter** [comment]
Ok cool. When I start the App I get this error:

![image](https://github.com/federicoReghini/vite-lib-reanimated/assets/2504695/d3f34577-2c6d-4178-a00e-6e4850d96e8b)

**Comment by federicoReghini** [comment]
Yes if you try to use it on web it is give this problem. But should be try it on ios or android. Do you think the problems are correlated?

**Comment by receter** [comment]
Not sure but finding out why this error comes up is a good idea anyway.

You could also try to copy the code directly to the project, then you at least know if it is related to packaging, if you know what I mean.

**Comment by FedeRBeije** [comment]
That was my first think too. Actually in my real project i'm doing that. I'm using  Comp and logic copy that i've inside my app instead of the one inside the library. At least until i will find a way to solve this problem

**Comment by federicoReghini** [comment]
That was my first think too. Actually in my real project i'm doing that. I'm using Comp and logic copy that i've inside my app instead of the one inside the library. At least until i will find a way to solve this problem
